state/migration_import.go

```diff
@@ -1814,12 +1814,20 @@ func (i *importer) remoteEntities() error {
 		src: i.model,
 		dst: i.st.db(),
 	}
+	offerUUIDByName := make(map[string]string)
+	for _, app := range i.model.Applications() {
+		for _, offer := range app.Offers() {
+			offerUUIDByName[offer.OfferName()] = offer.OfferUUID()
```

Comment:
Probably not a problem, as I don't fully understand the logic here, but is it possible to have multiple offer names with different UUIDs?

Reply:
No, a uuid is created for each offer and offer names are unique to a model.